### PR TITLE
Upgrade to Netty 4.1.43 & GraalVM 19.3.0 and fix support for Graal with JDK 11

### PR DESCRIPTION
**Pull request by ilopmar**
This PR should be merged with https://github.com/micronaut-projects/micronaut-profiles/pull/206

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -432,7 +432,7 @@ ext {
             ],
             graal                      : [
                     version: graalVersion,
-                    group  : 'com.oracle.substratevm',
+                    group  : 'org.graalvm.nativeimage',
```
Comment (jameskleeh) [inline build.gradle]: I don't think we can remove the old one without breaking existing apps

Comment (ilopmar) [reply build.gradle]: They have only changed the artifact coordinates but not the package for any class. Keep also in mind that this dependency is used as `compileOnly` when adding the `graal-native-image` feature in the CLI.
How do you suggest to do this change then?

File: build.gradle [hunk 2]
```diff
@@ -430,9 +430,14 @@ ext {
                     name   : 'hystrix-core',
                     modules: ['hystrix-serialization']
             ],
+            graalSubstrate             : [
```
Comment (sdelamo) [inline build.gradle]: why do we need `graalSubstrate`?. It does not seem to be used.

Comment (jameskleeh) [reply build.gradle]: To not break existing applications

**Pull request merged**